inc/msp/Client.hpp

```diff
@@ -126,111 +30,49 @@ enum MessageStatus {
 };
 
 struct ReceivedMessage {
-    uint8_t id;
-    std::vector<uint8_t> data;
+    uint32_t id;
+    ByteVector data;
     MessageStatus status;
 };
 
 class Client {
 public:
-    Client();
+    Client(const std::string &device, const size_t baudrate=115200);
```

Comment:
Move the device name and baudrate parameter back to the connection method. This way, you can attempt many connections with different settings without re-instantiating the class.